deny.toml

```diff
@@ -7,7 +7,7 @@ confidence-threshold = 0.93
 # Commented license types are allowed but not currently used
 allow = [
     "Apache-2.0",
-    # "BSD-2-Clause",
+    "BSD-2-Clause",
```

Comment:
What dependency are you pulling in that's new and licensed `BSD-2-Clause`? This is a surprising change.

Reply:
None of my changes require this "directly". The issue is that due to the absence of Cargo.lock, one of the dependencies is updating itself to the latest version and demanding this and the multiple-versions change for a successful build. Sean was forced to cherry pick the commit in his PR #48.